tooling/lsp/src/notifications/mod.rs

```diff
@@ -105,7 +75,12 @@ pub(super) fn on_did_close_text_document(
         state.cached_definitions.clear();
     }
 
-    ControlFlow::Continue(())
+    let document_uri = params.text_document.uri;
+
+    match process_workspace_for_noir_document(state, document_uri, true, false) {
```

Comment:
`let` bindings for multiple `bool`'s can be easier to read:
```suggestion
    let only_process_document_uri_package = true;
    let no_diagnostics = false;
    match process_workspace_for_noir_document(state, document_uri, only_process_document_uri_package, no_diagnostics) {
```

Reply:
Makes sense. Done!